docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```diff
@@ -769,77 +854,241 @@ back button label: Install a different version
 code: |
   install_task = background_action('install_alkiln', None, version=version_to_install)
 ---
+id: make npm dir if needed
+code: |
+  from pathlib import Path
+  try:
+    npm_dir = Path( "/var/www/.npm-global" )
+    npm_dir.mkdir( parents=True, exist_ok=True )
+    made_npm_dir_successfully = True
+  except Exception as make_npm_dir_error:
+    made_npm_dir_successfully = False
+    log(f"""🤕 ALKiln ERROR: Unable to make the directory "/var/www/.npm-global".""")
+---
```

Comment:
Consider moving this block below the block that depends on it, as we do for other such blocks.